matchms/importing/load_from_msp.py

```diff
@@ -14,7 +14,7 @@ def parse_msp_file(filename: str) -> Generator[dict, None, None]:
     # Peaks counter. Used to track and count the number of peaks
     peakscount = 0
 
-    with open(filename, 'r') as f:
+    with open(filename, 'r', errors='replace') as f:
```

Comment:
To me it is dangerous to replace things while reading the users input file without notifying the user. 
Do you have an example where this was needed?

Reply:
Yes. I got an unicode error when I tried to re-run a notebook from Efrain (student of Simon).
Here is the notebook: https://github.com/florian-huber/spec2vec_gcms_data_analysis/blob/master/notebooks/FH_re_run_pipeline_01.ipynb
The file (also present in the repo) is "MoNA-export-GC-MS.msp".

Reply:
Hmm, on a cloned repo I was able to read the file with `f = open('MoNA-export-GC-MS.msp').read()` with ipython 7.18.1 Python 3.8.5 n Ubuntu 20.04 using

```python

In [1]: 
   ...: import os
   ...: import sys
   ...: import numpy as np

In [2]: msp_file ="MoNA-export-GC-MS.msp"

In [3]: from matchms.filtering import normalize_intensities
   ...: from matchms.filtering import reduce_to_number_of_peaks
   ...: from matchms.filtering import require_minimum_number_of_peaks
   ...: from matchms.filtering import select_by_mz
   ...: from matchms.importing import load_from_msp

In [4]: z = list(load_from_msp(msp_file))

In [5]: len(z)
Out[5]: 14847

```
I did not get an unicode error, so I am unable to reproduce.

Reply:
I had it run on a Windows machine and get
``UnicodeDecodeError: 'charmap' codec can't decode byte 0x81 in position 135: character maps to <undefined>``

Reply:
I changed it to ``open(filename, 'r', encoding='utf-8')``